recipes/opencascade/all/conanfile.py

```diff
@@ -428,11 +431,7 @@ def _register_components(modules_dict):
                     system_libs = target_deps.get("system_libs", [])
                     frameworks = target_deps.get("frameworks", [])
 
-                    self.cpp_info.components[conan_component_target_name].names["cmake_find_package"] = target_lib
-                    self.cpp_info.components[conan_component_target_name].names["cmake_find_package_multi"] = target_lib
-                    self.cpp_info.components[conan_component_target_name].builddirs.append(self._cmake_module_subfolder)
```

Comment:
this will trigger https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h019-cmake-file-not-in-build-folders right ?

Reply:
No

Reply:
Well, take a look at the last lines in https://c3i.jfrog.io/c3i/misc/logs/pr/8869/1-configs/windows-visual_studio/opencascade/7.5.0//4eb25c38fccb68e6473f428d27ae7648cf3422c9-build.txt

Reply:
ok, so it' just a warning now, but this hook is wrong, you don't have to put cmake files listed in build_modules, in builddirs as well.

Reply:
I think we first have to make this discussion in https://github.com/conan-io/hooks/, and fix the hooks, before spreading this all over CCI

Reply:
https://github.com/conan-io/hooks/issues/375